Domains/3-DisciplineOther/StructuralAnalysis/Released/StructuralAnalysis.01.00.04.ecschema.xml

```diff
@@ -0,0 +1,1740 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- ==================================================================================
+|  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+|  * See LICENSE.md in the project root for license terms and full copyright notice.
+======================================================================================= -->
+<ECSchema schemaName="StructuralAnalysis" alias="sa" version="01.00.04" xmlns="http://www.bentley.com/schemas/Bentley.ECXML.3.2" displayLabel="Structural Analysis" description="Structural Analysis Schema.">
+    <ECSchemaReference name="CoreCustomAttributes" version="01.00.03" alias="CoreCA"/>
+    <ECSchemaReference name="BisCustomAttributes" version="01.00.00" alias="bisCA"/>
+    <ECSchemaReference name="BisCore" version="01.00.10" alias="bis" />
+    <ECSchemaReference name="Profiles" version="01.00.00" alias="prf" />
+    <ECSchemaReference name="Analytical" version="01.00.00" alias="anlyt" />
+    <ECSchemaReference name="AecUnits" version="01.00.03" alias="AECU" />
+
+    <ECCustomAttributes>
+        <ProductionStatus xmlns="CoreCustomAttributes.01.00.03">
+            <SupportedUse>NotForProduction</SupportedUse>
```

Comment:
SupportedUse should be set to "Production" on Released schemas.

Reply:
Fixed in https://github.com/iTwin/bis-schemas/pull/295/commits/a99b89b87533c4908755711b4019d5ffba62b216